javascript/javascript2/week1/danishLetters.js

```diff
@@ -0,0 +1,39 @@
+function countDanishLetters(wordString) 
+{
+     
+    let char;
+    let char1=0, char2=0, char3=0;
+
+    for (let i=0;i<wordString.length;i++)
+    {
+            char = wordString.slice(i,i+1);
+            
+            switch(char.toLowerCase())
+            {
+            case 'å':
+            char1++; 
+            break;
+            
+            case 'ø':
+            char2++; 
+            break;
+
+            case 'æ':
+            char3++; 
+            break;
+
```

Comment:
Missing a default case, even though you are not going to use it, it's still good practice to include it